src/test/java/org/minions/devfund/mauricioz/linkedList/RemoveMaximumValuesTest.java

```diff
@@ -0,0 +1,219 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+
+
+import java.util.LinkedList;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ // Source code recreated from a .class file by IntelliJ IDEA.
+ // (powered by Fernflower decompiler)
+ */
+public class RemoveMaximumValuesTest {
+    private static final int LISTSIZE1 = 3;
+    private static final int LISTSIZE2 = 4;
+    /**
+     * constructor.
+     */
+    public RemoveMaximumValuesTest() {
+    }
+
+    /**
+     * setup for tests.
+     */
+    @Before
+    public void setUp() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)